app/views/users/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<!-- <form action="/users" method="post" accept-charset="UTF-8">

```

Comment:
It's better to use `users_path` rails helper.

Reply:
This was a regular HTML form. We used actual action to test it and it is commented out.